response_operations_ui/templates/create-message.html

```diff
@@ -1,6 +1,7 @@
 {% extends "layouts/base.html" %}
+{% from "components/button/_macro.njk" import onsButton %}
 
-{% block page_title %} Create Message | Survey Data Collection {% endblock %}
+{% set page_title = "Create Message" %}
```

Comment:
Same thing with this page title.  This changes it doesn't it?

Reply:
As before, it's the same just done in a different way